src/Microsoft.Extensions.FileProviders.Abstractions/project.json

```diff
@@ -3,7 +3,7 @@
   "buildOptions": {
     "warningsAsErrors": true,
     "keyFile": "../../tools/Key.snk",
-    "xmlDoc": true
+    "xmlDoc": false
```

Comment:
I should probably revert this file. Perhaps making things `internal` in the other project would let me revert it in the second project too.